src/js/events/MetronomeClient.ts

```diff
@@ -2,33 +2,107 @@ import { request } from "@dcos/http-service";
 // TODO: remove this disable with https://jira.mesosphere.com/browse/DCOS_OSS-3579
 // tslint:disable-next-line:no-submodule-imports
 import { Observable } from "rxjs/Observable";
-
 import Config from "../config/Config";
 
 // Add interface information: https://jira.mesosphere.com/browse/DCOS-37725
-interface IJobData {
+export interface JobResponse {
+  id: string;
+  labels?: object;
+  run: {
+    cpus: number;
+    mem: number;
+    disk: number;
+    cmd: string;
+    env: object;
+    placement: {
+      constraints: any[];
+    };
+    artifacts: any[];
+  };
+  schedules: Schedule[];
+  historySummary: {
+    failureCount: number;
+    lastFailureAt: string | null;
+    lastSuccessAt: string | null;
+    successCount: number;
+  };
+}
+
+export interface JobDetailResponse {
   id: string;
+  description: string;
+  labels: object;
+  run: {
+    cpus: number;
+    mem: number;
+    disk: number;
+    cmd: string;
+    env: {};
+    placement: {
+      constraints: any[];
```

Comment:
I bet it's gonna be `string[]`, no?

Reply:
I don't know. We originally wanted to get the types from the RAML definitions, but that ticket is not in the sprint. If you see the commit, the primary idea here was to just move in some types and improve them while working on the respective resolvers.

Reply:
There is no RAML for Metronome, right? but in that case, it should be `string[]`.
Could we add a Jira to replace the any with something more meaningful in general?

Reply:
@Poltergeist There were some RAML definitions when I looked. From my point of view these are part of implementing the respective resolvers, so in this case writing the JobDetail resolver. If we need the data we type it. These were already in place, so I decided to add them. If that's something blocking I am fine with removing them from this PR